packages/dd-trace/src/llmobs/sdk.js

```diff
@@ -355,7 +381,7 @@ class LLMObs extends NoopLLMObs {
       timestamp_ms: timestampMs,
       tags: Object.entries(evaluationTags).map(([key, value]) => `${key}:${value}`)
     }
-
+    telemetry.recordSubmitEvaluation(options, '')
```

Comment:
```suggestion
    telemetry.recordSubmitEvaluation(options)
```
doing this in js is valid syntax and will define the second arg as `undefined` 😄